### PR TITLE
Add documentation about MSTest runsettings

### PR DESCRIPTION
**Pull request by Evangelink**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-configure.md](https://github.com/dotnet/docs/blob/50f09fe8746f45b4a9464158b8496780d6fa225e/docs/core/testing/unit-testing-mstest-configure.md) | [Configure MSTest](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-configure?branch=pr-en-us-40496) |

### CODE REVIEW

**Review by MarcoRossignoli — APPROVED** [review]

File: docs/core/testing/unit-testing-mstest-configure.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+|**MapInconclusiveToFailed**|false|If a test completes with an inconclusive status, it's mapped to the skipped status in **Test Explorer**. If you want inconclusive tests to be shown as failed, set the value to **true**.|
+|**MapNotRunnableToFailed**|true|A value indicating whether a not runnable result is mapped to failed test.|
+|**Parallelize**|Used to set the parallelization settings:<br /><br />**Workers**: The number of threads/workers to be used for parallelization, which is by default **the number of processors on the current machine**.<br /><br />**SCOPE**: The scope of parallelization. You can set it to **MethodLevel**. By default, it's **ClassLevel**.<br /><br />`<Parallelize><Workers>32</Workers><Scope>MethodLevel</Scope></Parallelize>`|
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-configure.md:39]:
```suggestion
|**Parallelize**||Used to set the parallelization settings:<br /><br />**Workers**: The number of threads/workers to be used for parallelization, which is by default **the number of processors on the current machine**.<br /><br />**SCOPE**: The scope of parallelization. You can set it to **MethodLevel**. By default, it's **ClassLevel**.<br /><br />`<Parallelize><Workers>32</Workers><Scope>MethodLevel</Scope></Parallelize>`|
```

File: docs/core/testing/unit-testing-mstest-configure.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+|**SettingsFile**||You can specify a test settings file to use with the MSTest adapter here. You can also specify a test settings file [from the settings menu](#specify-a-run-settings-file-in-the-ide).<br /><br />If you specify this value, you must also set the **ForcedLegacyMode** to **true**.<br /><br />`<ForcedLegacyMode>true</ForcedLegacyMode>`|
+|**TestCleanupTimeout**|0|Specify globally the timeout to apply on each instance of test cleanup method. `[Timeout]` attribute specified on the test cleanup method overrides the global timeout .|
+|**TestInitializeTimeout**|0|Specify globally the timeout to apply on each instance of test initialize method. `[Timeout]` attribute specified on the test initialize method overrides the global timeout .|
+|**TestTimeout**|0|Gets specified global test case timeout.|
+|**TreatClassAndAssemblyCleanupWarningsAsErrors**|false|To see your failures in class cleanups as errors, set this value to **true**.|
+|**TreatDiscoveryWarningsAsErrors**|false|To report test discovery warnings as errors, set this value to **true**.|
+
+### TestRunParameter element
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-configure.md:47]:
```suggestion
### `TestRunParameter` element
```

File: docs/core/testing/unit-testing-mstest-configure.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+</TestRunParameters>
+```
+
+Test run parameters provide a way to define variables and values that are available to the tests at run time. Access the parameters using the MSTest <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestContext.Properties%2A?displayProperty=nameWithType> property:
+
+```csharp
+private string _appUrl;
+public TestContext TestContext { get; set; }
+
+[TestMethod] // [Test] for NUnit
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-configure.md:61]:
```suggestion
[TestMethod] // [Test] for NUnit, [Fact] for xUnit
```

**Review by IEvangelist — APPROVED** [review]

**Comment by IEvangelist** [comment]
We need to address this MD lint error too:

> Error: In file 'docs/core/testing/unit-testing-mstest-configure.md': Invalid link: '#specify-a-run-settings-file-in-the-ide'.

**Pull request merged**